pkgs/by-name/le/legendsviewer-next/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  patches ? [ ],
+
+  lib,
+  dotnetCorePackages,
+  curl,
+
+  buildDotnetModule,
+  callPackage,
+  copyDesktopItems,
+  fetchFromGitHub,
+  makeDesktopItem,
+  nix-update-script,
+  writeShellScriptBin,
+}:
+let
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Kromtec";
+    repo = "LegendsViewer-Next";
+    tag = "v${version}";
+    hash = "sha256-Fi4KARGsgDmplH/oG9OIxY2XqhHNYgUB6OxwtoTaCt4=";
+  };
+
+  frontend = callPackage ./frontend.nix {
+    inherit version src patches;
+  };
+in
+buildDotnetModule rec {
+  pname = "legendsviewer-next";
+  inherit version src patches;
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "Legends Viewer";
+      genericName = "DF Legends Export Browser";
+      comment = meta.description;
+      icon = "${frontend}/dist/ceretelina.png";
+      tryExec = meta.mainProgram;
+      exec = meta.mainProgram;
+
+      # Remove when https://github.com/Kromtec/LegendsViewer-Next/pull/55 gets
+      # released
+      terminal = true;
+
+      categories = [
+        "Game"
+        "RolePlaying"
+        "Simulation"
+      ];
+      keywords = [
+        "df"
+        "dwarf"
+        "fortress"
+        "legends"
+        "viewer"
+      ];
+    })
+  ];
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+  nugetDeps = ./deps.json;
+  projectFile = "./LegendsViewer.Backend/LegendsViewer.Backend.csproj";
+  testProjectFile = "./LegendsViewer.Backend.Tests/LegendsViewer.Backend.Tests.csproj";
+  executables = [ meta.mainProgram ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    # Fixes a build failure, the frontend node build is handled in ./frontend.nix
+    (writeShellScriptBin "npm" "")
```

Comment:
Can we patch the code out instead? Right now we also would void future invocations of npm which we might or might not want to void